cloud/linode/loadbalancers.go

```diff
@@ -558,6 +558,11 @@ func (l *loadbalancers) EnsureLoadBalancerDeleted(ctx context.Context, clusterNa
 		return nil
 	}
 
+	fwClient := firewall.LinodeClient{Client: l.client}
+	if err = fwClient.DeleteNodeBalancerFirewall(ctx, service, nb); err != nil {
```

Comment:
sorry i missed this earlier 😅 
```suggestion
	if err := fwClient.DeleteNodeBalancerFirewall(ctx, service, nb); err != nil {
```

Reply:
err var was defined previously at line 536 https://github.com/linode/linode-cloud-controller-manager/blob/remove-fw/cloud/linode/loadbalancers.go#L536 , so should be fine with = instead of := . Do we really need : here?

Reply:
not a big deal - but its good to scope the err's within the if, especially if we're returning right after checking that - this is to avoid shadowing stuff if we have defer's (Which we don't)

Reply:
I see, yeah that makes sense.